packages/autorest.typescript/test/commands/run.ts

```diff
@@ -136,7 +140,7 @@ export async function runAutorest(
 
   commandArguments.push(
     inputFileCommand,
-    "--version=3.6.6",
+    "--version=3.9.3",
```

Comment:
I found if I don't bump this autorest core version, the ci will just hang in there and CI will keep get timeout error